manim/cli/render/commands.py

```diff
@@ -136,4 +139,22 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if nov or args["notify_outdated_version"]:
+        manim = requests.get("https://pypi.org/pypi/manim/json")
+        releases = manim.json()["releases"].keys()
+        latest = max(releases)
+
+        if latest != __version__:
+            console.print(
+                "[yellow]WARNING: You are running on an outdated version of manim[/yellow]"
+            )
+            console.print(f"Installed version: [red]v{__version__}[/red]")
+            console.print(f"Latest version: [green]v{latest}[/green]")
+            console.print(
```

Comment:
I think that this is a good feature until right here. This print (along with the one below it with the link) are longer and more obtrusive than just always displaying if manim is outdated. If it were me receiving the warning, I'd probably keep it around just so that I don't forget I'm not on the most recent version, but because of the long message, I'd probably turn it off.

Reply:
I'm pretty sure I clicked the right line, but oh well, _github_. Since it isn't clear what I am talking about, I am referencing notifying the user how to turn off the notification (this seems like a weird UX choice anyway).

Reply:
I guess the installed/latest version could be combined into a single line similar to how pip does it. Taking from stackoverflow:
> You are using pip version 6.0.8, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

I think the above example with pip should be translated to manim as:
>You are using manim version A.B.C, however version D.E.F is available.
You should consider upgrading via `pip install -U manim`.

Additionally, pip also has a config file setting to disable/quiet this warning, but they don't mention it. I think it's fine to keep the warning but I'm not sure about the link to the latest documentation. Again, `latest != most recently released`

Reply:
I just think it's counterproductive to notify a user on how to turn off another notification. It takes far more space in the terminal than just mentioning you're outdated. And if it's combined into one line, I especially don't see a reason for it. We could remove this message and just mention it in our docs so that if someone searches how to disable the warning (which I already doubt will happen if it's a small warning), they'll find mention of it in our docs.